regression/contracts/replace-nondet-return-value/main.c

```diff
@@ -0,0 +1,23 @@
+int cmp(int i1, int i2)
```

Comment:
Random place for comment: can you please add "Fixes: #6483" to the commit message so that the issue is auto-resolved once this PR is merged?

Reply:
I'll go ahead and merge anyway, so this is just for future reference.